modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/AM/configFiles/approveWorkflow/workflow-extensions.xml

```diff
@@ -47,4 +47,9 @@
         <Property name="stateList">Created:Publish,Published:Block</Property>
     </APIStateChange>
 
+    <!--APIProductStateChange executor="org.wso2.carbon.apimgt.impl.workflow.APIProductStateChangeSimpleWorkflowExecutor" /-->
```

Comment:
Shall we remove this?

Reply:
This is the default executor for API Products. We have the default lifecycle executors commented out in this way for other flows too.